addons/website_sale/models/product_template.py

```diff
@@ -295,7 +295,7 @@ def _get_previewed_attribute_values(self):
                 if len(previewed_ptavs) > 1:
                     previewed_ptavs_data = []
                     for ptav in previewed_ptavs[:show_count]:
-                        matching_variant = ptav.ptav_product_variant_ids.sorted('id')[0]
+                        matching_variant = ptav.ptav_product_variant_ids and ptav.ptav_product_variant_ids.sorted('id')[0]
```

Comment:
```suggestion
                        matching_variant = ptav.ptav_product_variant_ids.sorted('id')[:1]
```
wouldn't it be simpler like that?
Would also guarantee a consistent variable type